### ISSUE TITLE
regularize new-style focus indications 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#364]
By way of prior tickets (especially #245), we have implemented a new, highly visible focus indication involve blue linework. As @alecpm [pointed out](https://github.com/isawnyu/isaw.web/issues/245#issuecomment-445132069), we should extend this approach to include:

 - logo
 - search button
 - navigation portlet
 - footer

**Comment by alecpm** [comment]
I've implemented all of these on staging. For the footer links and links in the slider overlay, I had to choose a new color that had better contrast with the dark background than the blue, so I picked a light orange #FFB00F with a 7:1 contrast ratio. Let me know if you'd like to change that though.

**Comment by alecpm** [comment]
Oh right, I'll have to update the width to include padding + border.

**Comment by alecpm** [comment]
Should be fixed now.

**Comment by paregorios** [comment]
Yes! Please merge to master and deploy to production!

**Comment by paregorios** [comment]
Looks good on production.
